style.css

```diff
@@ -1,6 +1,10 @@
 /* stylelint-disable no-descending-specificity */
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap");
+
 body {
   background-color: #fff;
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
 }
 
 .flex-container {
```

Comment:
- [ ] As stated in the project requirement In order to lay out the elements on the page you should use `Flexbox` and You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your design Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/200660643-d8063c47-8349-481e-95df-188e46dd0f17.PNG)

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/200660814-e1002762-fc53-4baa-8811-4d9c9f8e0992.PNG)

Reply:
Fixed